mailbox/store/src/main/java/org/apache/james/mailbox/store/mail/model/impl/MessageParser.java

```diff
@@ -73,17 +73,18 @@
     private static final Logger LOGGER = LoggerFactory.getLogger(MessageParser.class);
 
     private final Cid.CidParser cidParser;
+    private final DefaultMessageBuilder defaultMessageBuilder;
 
     public MessageParser() {
         cidParser = Cid.parser()
             .relaxed()
             .unwrap();
+        defaultMessageBuilder = new DefaultMessageBuilder();
```

Comment:
i'm not comfortable with that kind of change, DefaultMessageBuilder happens to (likely) be thread safe and only stores configuration as state but it's not even in its documentation.
Do we gain anything with this change ? If we want to keep that, can we add documentation on DefaultMessageBuilder to express the thread-safe and stateless properties ?

Reply:
I'll remove that commit